packages/manager/modules/pci/src/projects/new/new.controller.js

```diff
@@ -256,7 +257,16 @@ export default class PciProjectNewCtrl {
   ============================== */
 
   onNextBtnClick() {
+    const page = `public-cloud::${this.$state.current.name.replace(/\./g, '::')}`;
```

Comment:
`$state` is not injected in controller but in routing file

Reply:
right!